parsing_input.c

```diff
@@ -0,0 +1,61 @@
+#include "shell.h"
```

Comment:
There should be a space before the comment, otherwise betty would be angry with the source code